### PR TITLE
Bug 1656195 - Repository hyperlink from Graphs view does not show related tests from other repositories

### PR DESCRIPTION
**Pull request by esanuandra**
[Bug 1656195 - Repository hyperlink from Graphs view does not show related tests from other repositories](https://bugzilla.mozilla.org/show_bug.cgi?id=1656195)

### CODE REVIEW

File: ui/perfherder/graphs/TestDataModal.jsx [hunk 1]
```diff
@@ -220,7 +219,7 @@ export default class TestDataModal extends React.Component {
       })
       .filter(
         (item) =>
```
Comment (ionutgoldan) [inline ui/perfherder/graphs/TestDataModal.jsx:221]: `item` is very generic & doesn't give any immediate clue on what it's holding. Could you give it a better name?

File: ui/perfherder/graphs/TestDataModal.jsx [hunk 2]
```diff
@@ -220,7 +219,7 @@ export default class TestDataModal extends React.Component {
...
-          item.name === relatedSeries.name &&
+          item.name === relatedSeries.name.trim() &&
```
Comment (ionutgoldan) [inline ui/perfherder/graphs/TestDataModal.jsx:222]: I am able to retrieve related series without needing to `trim()`.
Are you sure this fix is needed? Could you share the Graph view links you've been using to test this?

Comment (esanuandra) [reply ui/perfherder/graphs/TestDataModal.jsx:222]: http://localhost:5000/perfherder/graphs?timerange=5184000&series=mozilla-central,2833204,1,6

Comment (ionutgoldan) [reply ui/perfherder/graphs/TestDataModal.jsx:222]: Thanks for the link. Looks like we need this `trim()` operation because of a backend bug.

Comment (ionutgoldan) [reply ui/perfherder/graphs/TestDataModal.jsx:222]: @esanuandra could you actually trim both names?

File: ui/perfherder/graphs/TestDataModal.jsx (hunk 1 above)
Comment (esanuandra) [reply ui/perfherder/graphs/TestDataModal.jsx:221]: `responseSeries` would be better?

File: ui/perfherder/graphs/TestDataModal.jsx (hunk 2 above)
Comment (esanuandra) [reply ui/perfherder/graphs/TestDataModal.jsx:222]: I suspected as well the issue might be linked to Bug 1523037.
Sure, I will trim them both.

File: ui/perfherder/graphs/TestDataModal.jsx (hunk 1 above)
Comment (ionutgoldan) [reply ui/perfherder/graphs/TestDataModal.jsx:221]: Yes, it is better.

**Review by ionutgoldan — APPROVED** [review]
Looking good!

**Pull request merged**